### ISSUE TITLE
Feature: Highlight text in Request Body

### ISSUE THREAD

**Issue by rwjack** [healthchecks/healthchecks#710]
Sometimes there's too much text to sift through in the Request Body of a check.

Perhaps it would be a cool feature to have some sort of syntax highlighting?

An idea is to have an option in settings for setting the default syntax (default syntax could be per check/project/account?), and also having a dropdown box to chose syntax highlighting language (bottom right of the image - in line with "Request Body")
![image](https://user-images.githubusercontent.com/59068073/192877835-83dc776f-51fe-4beb-8563-560886f765d9.png)

**Comment by cuu508** [comment]
Hi @rwjack, what syntax formats do you have in mind? JSON, XML, something like that?

**Comment by rwjack** [comment]
Yeah I'm trying real hard, but I can't seem to remember where I was going this idea.

Anyways, most of my output isn't syntax, it's just basic echo commands with text, so not sure how highlighting that would work.

Is there some sort of standard for text output highlighting? Like "Logging syntax" or something?

In that case - to solve this idea:
- users could modify their scripts to use a logging standard
- healthchecks could have an option to highlight that type of logging syntax (also including options for JSON, XML, YAML - basically the most commonly used output languages)

**Comment by rwjack** [comment]
@cuu508 Hey, I think I remembered why I opened this initially.

It's because I use runitor + healthchecks + ansible as an alternative to ansible tower, so I usually have ansible-syntax type output in my checks.

ansible + terminal = syntax highlighted

ansible + terminal + runitor = no highlighting

Since most of the jobs run as cron, there is no need to push for runitor to show the colors of the program it's running (though it would be a nice feature), but it would be great if healthchecks would color code the output somehow.

Let me know what you think!
